main.py

```diff
@@ -75,7 +75,7 @@ def run_analysis(input_type, user_input):
         "Please note that this estimation does not need to be correct and should be further supported by the in-depth "
         "analysis below.")
     st.subheader("Topic Evolution Analysis:")
-    image = te.graph().pipe(format='jpg')
+    image = get_default_te_graph(content).pipe(format='jpg')
```

Comment:
We are running tem twice on the same content here in the same function right? 
I think it would be great to have a method that returns pair(tem, metrics) and then a get_te_graph(tem) instead of using content again. This would probably speed up result generation for users.

Reply:
yes, we are. TEM can currently only return either the graph or the metrics, so we have to run the model twice. we can change this at some point but I would argue it's not the highest priority right now. feel free to open an issue on the TEM repo :)